kpet/data.py

```diff
@@ -313,12 +320,23 @@ def matches(self, target):
         Returns:
             True if the case matches the target, False otherwise.
         """
-        return self.pattern.matches(target)
+        if target.sets is Target.ANY or target.sets is Target.ALL or\
```

Comment:
I would say `==`, not `is`, would be more appropriate here.

Reply:
Yeah I was replacing a None there so I wasn't sure if to change it or not